backend/sirius-web-diagrams-layout/src/main/java/org/eclipse/sirius/web/diagrams/layout/incremental/updater/OverlapsUpdater.java

```diff
@@ -117,6 +117,9 @@ private Position getCenter(NodeLayoutData node) {
 
     private Position translate(Position source, Position target, double distance, Position start) {
         double factor = Math.sqrt(Math.pow(source.getX() - target.getX(), 2.0) + Math.pow(source.getY() - target.getY(), 2.0));
+        if (factor == 0) {
+            factor = 1;
+        }
```

Comment:
Seems to work (tested in jshell), but comparing to `0.0d` and setting the value to `1.0d` would make it more explicit that  we're dealing with `double`s here.

Reply:
Done